core/Pos/Binary/Core/Types.hs

```diff
@@ -61,3 +61,32 @@ instance Bi T.SharedSeed where
 instance Bi T.ChainDifficulty where
     get = label "ChainDifficulty" $ T.ChainDifficulty . getUnsignedVarInt <$> get
     put (T.ChainDifficulty c) = put (UnsignedVarInt c)
+
+instance Bi T.BlockVersionData where
+    get = label "BlockVersionData" $ do
```

Comment:
There are two separate issues here and i'm not sure which one we're talking about:
* What neongreen talks about: explicit serialization. AKA "not using deriving". 
* What possibly pva228 is talking about: default instances/methods backed up by `Binary`. IDK. 
I don't really understand though why isn't it better to do
```
get = T.BlockVersionData <$> get <*> ... <*> get
put BlockVersionData{..} = forM_ [bvdA, bvdB...] put
```
Would save us some space 🤷‍